### PR TITLE
Fix #199:  of types results in incorrect output

### LINKED ISSUE #199: `#define` of types results in incorrect output
eg:

`test.h`:
```c
#define __le64 uint64_t;
```

dstep output:
`test.d`:
```d
enum __le64 = uint64_t;
``` 

tested on macOS 10.13 with dstep master

### PR DESCRIPTION
**Pull request by ciechowoj**

### CODE REVIEW

File: dstep/translator/MacroDefinition.d [hunk 1]
```diff
@@ -579,22 +579,32 @@ string spelling(Expression expression)
...
     auto expressionContext = ExpressionContext.make(context);
 
-    version (D1)
-        enum formatString = "const %s = %s;";
+    string formatString;
+    auto debraced = definition.definition.expr.debraced;
+
+    if (debraced.peek!TypeIdentifier)
+    {
+        formatString = "alias %s = %s;";
```
Comment (jacob-carlborg) [inline dstep/translator/MacroDefinition.d]: Not sure if we need to bother but this will only work with D2. @mihails-strasuns?

Comment (ciechowoj) [reply dstep/translator/MacroDefinition.d]: How was it solved in D1? `alias old_name_t new_name_t;` ?

File: dstep/translator/MacroParser.d [hunk 2]
```diff
@@ -1464,18 +1464,18 @@ class MacroDefinition
     Cursor cursor;
     string spelling;
     string[] params;
-    bool constant;
+    bool alias_or_const;
```
[the remark this answers is not in the document]
Comment (ciechowoj) [reply dstep/translator/MacroParser.d]: My mistake.

[the remark this answers is not in the document]
Comment (ciechowoj) [reply dstep/translator/MacroParser.d]: Fixed.

File: dstep/translator/MacroDefinition.d (hunk 1 above)
Comment (jacob-carlborg) [reply dstep/translator/MacroDefinition.d]: Yes.

Comment (ciechowoj) [reply dstep/translator/MacroDefinition.d]: Updated, but I'll leave it without test. No idea how to even test it as the `version (D1)` is compile-time statement...

Comment (jacob-carlborg) [reply dstep/translator/MacroDefinition.d]: That's fine.

**Pull request merged**